analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -601,10 +607,23 @@ def parse_with_plt_formatter(plist_file: str,
     return changed_files
 
 
-def parse_convert_reports(input_dirs: List[str],
-                          out_format: str,
-                          severity_map: Dict,
-                          trim_path_prefixes: List[str]) \
+class TrimPathPrefixHandler:
+    def __init__(self, prefixes: Optional[List[str]]):
+        self.__prefixes = prefixes
+
+    def __call__(self, source_file_path):
+        """
+        Callback to util.trim_path_prefixes to prevent module dependency
+        of plist_to_html
+        """
+        return util.trim_path_prefixes(source_file_path, self.__prefixes)
+
+
+def _parse_convert_reports(input_dirs: List[str],
+                           out_format: str,
+                           severity_map: Dict,
+                           trim_path_prefixes: List[str],
+                           skip_handler: Callable[[str], bool]) \
```

Comment:
A couple of places I used the following code style:
```suggestion
def _parse_convert_reports(
    input_dirs: List[str],
    out_format: str,
    severity_map: Dict,
    trim_path_prefixes: List[str],
    skip_handler: Callable[[str], bool]
) -> Tuple[Union[Dict, List], int]:
```
In my opinion it is better because we have more free spaces, so if a variable name / type is too long for a parameter we don't have to break long lines, so the code will be much more readable.

So if you agree we can use this style to declare a function / parameter. In this case plase change the style in other places as well.